projects/addon-editor/directives/tiptap-editor/tiptap-editor.directive.ts

```diff
@@ -35,21 +38,36 @@ export class TuiTiptapEditorDirective {
     }
 
     @Output()
-    readonly valueChange = this.editor.valueChange$;
+    readonly valueChange = this.editor.valueChange$.pipe(this.debounceChanges());
 
     @Output()
-    readonly stateChange = this.editor.stateChange$;
+    readonly stateChange = this.editor.stateChange$.pipe(this.debounceChanges());
```

Comment:
Why we can't just use [debounceTime](https://rxjs.dev/api/index/function/debounceTime) ?

```ts
.pipe(debounceTime(this.options.emitUpdateTime)
```

Reply:
Than that example will be broken

<img width="854" alt="image" src="https://github.com/Tinkoff/taiga-ui/assets/12021443/989e68b8-03d4-4c0a-b177-3362f9a7cfa7">